molo/core/tasks.py

```diff
@@ -25,18 +26,40 @@ def rotate_content():
     site = Site.objects.get(is_default_site=True)
     settings = SettingsProxy(site)
     site_settings = settings['core']['SiteSettings']
-    if site_settings.content_rotation and \
-            site_settings.content_rotation_time == datetime.now().hour:
-        if main and index:
+    if main and index:
+        if site_settings.content_rotation_time == datetime.now().hour:
+            if site_settings.content_rotation:
+                rotate_latest(main_lang, index, main)
+            rotate_featured_in_homepage(main_lang)
```

Comment:
@miltontony It shouldn't. Rotate latest only rotates if rotate content is set. featured in homepage rotates if the section rotation is set. That get's checked in its task